src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+enum Colors {
```

Comment:
```suggestion
public enum Color {
```
remember about access modifier, class names should not be named in plural